wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -62,10 +77,11 @@ public static HPCCWsWorkUnitsClient get(Connection connection)
 
     public static final String          WSWORKUNITSWSDLURI          = "/WsWorkunits";
     private WsWorkunitsServiceSoapProxy wsWorkunitsServiceSoapProxy = null;
+    private org.hpccsystems.ws.client.gen.wsworkunits.v1_56.WsWorkunitsServiceSoapProxy fallBackWorkunitsServiceSoapProxy =null;
     public static final int             defaultWaitTime             = 10000;
     public static final int             defaultResultLimit          = 100;
     public static final int             defaultMaxWaitTime          = 1000 * 60 * 5;
-
+    private Version						version						= null;
```

Comment:
let's make the var name more self-documenting, also stylistic comment let's try to keep the tab/space uniform